test/js/search/study.test.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import * as ReactAll from 'react';
+import { mount } from 'enzyme';
+
+const fetch = require('node-fetch');
+
+import Study from 'components/Study';
+
+const facetMatchedStudy = {
+  name: 'Tuberculosis subtypes in human male blood',
+  description: 'stuff and things',
+  cell_count: 130,
+  facet_matches: {
+    facet_search_weight: 1,
+    organ: [{id: 'id1', name: 'blood'}]
+  }
+}
+
+const complexFacetMatchedStudy = {
+  name: 'Tuberculosis subtypes in human male blood',
+  description: 'stuff and things',
+  cell_count: 130,
+  facet_matches: {
+    facet_search_weight: 1,
+    organ: [{id: 'id1', name: 'blood'}],
+    species: [{id: 'id5', name: 'mouse'}, {id: 'id7', name: 'human'}]
+  }
+}
+
+const facetUnmatchedStudy = {
+  name: 'Tuberculosis subtypes in human male blood',
+  description: 'stuff and things'
+}
+
+describe('Facet match badges', () => {
+  it('renders no badges with no matches', async () => {
+    const wrapper = mount((
```

Comment:
I don't know if mount is needed here. Mount does a full rendering including child components. In this case we're not testing child behavior so I don't believe mount is necessary.  I think render or shallow would be better. It would also increase the speed of the test because a full render isn't being performed.